client/package.json

```diff
@@ -10,8 +10,14 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "@reduxjs/toolkit": "^1.9.5",
+    "i": "^0.3.7",
```

Comment:
what is this? did you add this by mistake?
if yes - please remove it

Reply:
yes it by mistake , i will remove it